features/keymap/ncl/layers-default.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Layers (Default Layer)
+
+  The `K.layer_mod.set_default` key allows setting the Default layer.
+
+  Background:
+
+    Given a keymap.ncl:
+      """
+      let K = import "keys.ncl" in
+      {
+        layers = [
+          [
+            K.layer_mod.set_default 0,
+            K.layer_mod.set_default 1,
+            K.A,
+          ],
+          [
+            K.TTTT,
+            K.TTTT,
+            K.B,
+          ],
+        ],
+      }
+      """
+
+  Example: tapping the set default layer modifier key changes the default layer
+
+    When the keymap registers the following input
+      """
+      [
+        press (K.layer_mod.set_default 1),
+        release (K.layer_mod.set_default 1),
+        press (K.B),
```

Comment:
1. "tap" rather than "press", "release" may be nicer.

2. "press K.B" is rather hard to read in this case.